openmetadata-service/src/main/java/org/openmetadata/service/resources/teams/UserResource.java

```diff
@@ -692,6 +695,7 @@ public Response createOrUpdateUser(
           securityContext,
           getResourceContextByName(user.getFullyQualifiedName()),
           new OperationContext(entityType, MetadataOperation.CREATE));
+      user.withLastLoginTime(System.currentTimeMillis());
```

Comment:
Is this recording the user's last login time? Not sure it makes sense since in a scenario where someone else (like an admin) creates the user but the user themself do not log in and create a session.

Reply:
@sushi30 updated the PR, it should be there in @Post and if selfSignUp is enabled , thanks for the review